src/styles/main.scss

```diff
@@ -1,3 +1,51 @@
-body {
+/* Styles go here */
+@use "./blocks/header.scss";
+@use "./blocks/nav.scss";
+@use "./blocks/card.scss";
+@use "./blocks/stars.scss";
+@use "./utils/variables";
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+* {
   margin: 0;
+  padding: 0;
```

Comment:
shouldn't remove margins and paddings globally instead remove them on specific elements